pkg/api/types.go

```diff
@@ -1660,7 +1660,12 @@ func (o *OrchestratorProfile) NeedsExecHealthz() bool {
 		!common.IsKubernetesVersionGe(o.OrchestratorVersion, "1.9.0")
 }
 
-// HasAadProfile  returns true if the has aad profile
+// GetPodInfraContainerSpec returns the sandbox image as a string (ex: k8s.gcr.io/pause-amd64:3.1)
+func (o *OrchestratorProfile) GetPodInfraContainerSpec() string {
```

Comment:
This looks Linux specific, can you put Linux in the func name? We'll need a similar one for Windows later

Reply:
Should we not just return the proper image based on the orchestrator profile? I would imagine this func could provide back a Windows based image with the information currently provided to it.

Reply:
if we move to the multi-arch images (mcr.microsoft.com/kubernetes/pause:...) then it will work on all machines as-is. Otherwise we need to decide based off each node pool.

Reply:
it either needs to differ by node, or be a multi-arch multi-os image. `mcr.microsoft.com/k8s/pause` is such image. What's the default for `KubernetesConfig.KubernetesImageBase` ? If it matches then this is ok. the gcr images don't work with Windows

Reply:
Great point about the node pools. I hadn't thought about that. I'll change the PR to use `mcr.microsoft.com/k8s/pause`.

Thanks for the great review!!

Reply:
`mcr.microsoft.com/k8s/pause` was not correct. The correct image uri is `mcr.microsoft.com/k8s/core/pause`. I've standardized on tag `1.2.0`.